selfdrive/car/hyundai/values.py

```diff
@@ -348,35 +344,66 @@ class Buttons:
 }
 
 
-def get_platform_codes(fw_versions: List[bytes]) -> Set[bytes]:
-  codes: DefaultDict[bytes, Set[Optional[bytes]]] = defaultdict(set)
+def get_platform_codes(fw_versions: List[bytes]) -> Set[Tuple[bytes, Optional[bytes]]]:
```

Comment:
can you make a debug script that lives in hyundai/tests that prints out the list of platforms and their platform codes?

Reply:
https://github.com/commaai/openpilot/pull/28549